Swift.sublime-syntax

```diff
@@ -133,7 +145,7 @@ contexts:
       captures:
         1: storage.type.function
         2: entity.name.function variable.function
-    - match: \b(func|deinit|init)\b
+    - match: \b(func|deinit|init(\\?))\b
```

Comment:
`\\?` should be `\??`. If you don’t get to this before I do, I can merge and fix it and then push up a new tag.